test/spec/git/fixtures/async-blob.json

```diff
@@ -1,7 +1,7 @@
 {
 	"sha": "5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1",
 	"size": 5060,
-	"url": "https://api.github.com/repos/borisyankov/DefinitelyTyped/git/blobs/5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1",
+	"url": "https://api.github.com/repos/DefinitelyTyped/DefinitelyTyped/git/blobs/5ba56e0dbb7cc433d5b98ab3cbbf5e0d92954ff1",
```

Comment:
It is not related to my fix, just for passing tests on CI.
(but some errors occurred because of rate limit exceed... I've confirmed passing tests on a local machine using my github token.)